plutus-core-spec/figures/TypeSynthesis.tex

```diff
@@ -202,12 +202,13 @@
 
```

Comment:
I don't think this file is used.  The typing rules are included directly in plutus-core-specification.tex now. I'm nit sure how that happened.